src/templates/layouts/vbaFacility/index.tsx

```diff
@@ -188,6 +189,29 @@ export function VbaFacility({
               </va-alert>
             )}
             <div>TODO: Add conditional get updates links</div>
+
+            {ccGetUpdates && (
+              <section className="vads-u-background-color--gray-lightest vads-u-margin-top--4 mobile-lg:vads-u-margin-top--6 vads-u-padding-x--3 vads-u-padding-y--2p5">
```

Comment:
I wonder if it would be worth creating a dumb component that is used by both this and https://github.com/department-of-veterans-affairs/next-build/blob/main/src/templates/layouts/healthCareLocalFacility/FacilitySocialLinks.tsx so they'll have consistent markup and styles. Something like 

```jsx
<GetUpdatesSection heading={ccGetUpdates.heading} links={ccGetUpdates.links}} />
```

Reply:
I actually am using that component I linked to on the `VamcSystem` page too, but I never moved it or renamed it to show that it's shared.

Reply:
Yes! I actually had it in a component for a bit and then refactored it out since it ended up being simple and I didn't notice the other similar pattern. I'll refactor that tomorrow

Reply:
Ha ha, nice!

Reply:
Update: We chatted offline, and I see that it's not that simple. The other one has complicated logic for which things go in which columns, and that complexity is prohibitive to sharing right now.